apps/web/components/ui/UsernameAvailability/UsernameTextfield.tsx

```diff
@@ -168,7 +168,17 @@ const UsernameTextfield = (props: ICustomUsernameProps & Partial<React.Component
                   <p className="text-subtle" data-testid="new-username">
                     {t("new_username")}
                   </p>
-                  <p className="text-emphasis mt-1 break-all">{inputUsernameValue}</p>
+
+                  <p
+                    className="text-emphasis mt-1 break-all"
+                    style={{
+                      overflowY: "auto",
```

Comment:
Instead of using the styles attribute here why not pass in those directly as tailwind classnames? eg: `className="text-emphasis mt-1 break-all overflow-y-auto"` something like that